client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -28,6 +27,9 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 	if ( displayJetpackAppBranding ) {
 		const showIosBadge = ! isAndroid && ! isDesktop;
 		const showAndroidBadge = ! isIos && ! isDesktop;
+		const showBadge = showIosBadge || showAndroidBadge;
+
+		const badgeStoreName = showIosBadge ? 'ios' : 'android';
```

Comment:
Nit: since this value is only used once, inlining the conditional on line 53 rather than assigning a constant here might improve clarity.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/69000/commits/089f5b047af19b8ad5e865f5f4c0d7ee66773da6, thanks for bringing that perspective!